added Mac, Win install steps to Toolbox, and links btwn 3-in-1 getsta…

londoncalling opened the pull request:
…rted tutorial

Signed-off-by: Victoria Bialas <victoria.bialas@docker.com>

londoncalling commented:
@SvenDowideit @thaJeztah there are a few unresolved links here which should be resolved when the getting started tutorial is merged. That PR is coming up, ready to be merged next.

SvenDowideit commented on docs/faqs/index.md:
```diff
@@ -0,0 +1,20 @@
+description = "FAQs, troubleshooting, and tips index for Toolbox installs"
+keywords = ["docker, documentation, about, technology, kitematic, gui, toolbox"]
+identifier = "toolbox_overview_faqs"
++++
```
not sure - what menu should this go in - will fix after merge.

SvenDowideit commented:
```
04:04:27 # Filtered (toolbox) Summary:
04:04:27 
04:04:27 * link error: (in page toolbox/overview.md) /engine/tutorials/
04:04:27 * link error: (in page toolbox/overview.md) /engine/getstarted/
04:04:27 
04:04:27 	Found: 793 files
04:04:27 	Found: 2 errors
```

I'll fix that in a follow on PR

SvenDowideit commented:
LGTM once the 2 links are fixed.

SvenDowideit commented:
and https://github.com/docker/docker/pull/23565 goes with it.

londoncalling commented:
@SvenDowideit links are fixed I believe, but it turns out I do not seem to have merge permissions on Toolbox. @nathanleclaire @jeanlaurent can one of you all merge this for us?

@SvenDowideit, Per the Troubleshooting topic, we currently have this in the Toolbox menu. I do think you should probably remove the original "tutorials" for v1.12 since I've moved/munged the Getting Started tutorial(s) into repo `docker/docs/engine/getstarted/` and added the relevant Toolbox troubleshooting docs into Toolbox. Was there anything else in those tutorials docs being used?

FrenchBen commented:
@londoncalling Looks like documentation is failing - is this normal?

londoncalling commented:
@FrenchBen it looks like a test is failing but no link errors in the current docs. I have another PR open for Getting Started tutorial in `docker/docker` that should fix any related problems between these two documents, but this _should_ be good. This PR fixed several links and other problems, in addition to content.

londoncalling commented:
@dgageot can you merge this PR to help fix CI builds?

dgageot commented:
@londoncalling done
